dallinger/command_line/docker_ssh.py

```diff
@@ -24,6 +24,8 @@
 
 import click
 import requests
+import yaml
```

Comment:
YAML is a superset of JSON: every JSON file is also a valid YAML file with the same content.

Insted of adding a dependency I think we could produce a JSON file, and call it YAML.

Dallinger already depends on many packages, so it's better to be conservative about it.